gdk-pixbuf/src/pixbuf.rs

```diff
@@ -313,6 +313,9 @@ impl Pixbuf {
     pub unsafe fn pixels(&self) -> &mut [u8] {
         let mut len = 0;
         let ptr = ffi::gdk_pixbuf_get_pixels_with_length(self.to_glib_none().0, &mut len);
+        if len == 0 {
```

Comment:
`ptr` could be NULL.

Reply:
No it can't be `NULL` without `len` being `0`.

Reply:
Before adding more comments like this, please check the C API behaviour